blog-posts/step-function-learnings/step-function-logs-learning.md

```diff
@@ -0,0 +1,88 @@
+---
+published: false
+title: "Stop looking for your express state machine logs, they don't exist"
+description: 'Logs for express Step Function with the CDK'
+tags: logs, stepfunction, serverless, cdk
+series:
+canonical_url:
+---
+
+In this article you will learn how to add **logs in an express step function** with the CDK because it is not enabled by default. When you go from a standard step function with logs to an express one that gives you nothing.
+
+TL;DR
+
+## Standard workflows have logs enabled by default inside AWS Step Function
+### Read this to better grasp what is happening with those log groups in Step Function or skip to the next section for the tutorial you came for
+
+AWS explains it really clearly in its documentation, here is a quote describing the default logs behaviors of both Standard and Express types of State Machine.
+
+>Standard Workflows record execution history in AWS Step Functions, although you can optionally configure logging to Amazon CloudWatch Logs.
+>
+>Unlike Standard Workflows, Express Workflows don't record execution history in AWS Step Functions. To see execution history and results for an Express Workflow, you must configure logging to Amazon CloudWatch Logs. Publishing logs doesn't block or slow down executions.
+
+In the following images, you can see that for three different State Machines one Express configured with logs, one Express configured without logs and one Standard, the only one that has a log group associated is the Express one configured with a log group.
```

Comment:
three different state machines: one configured...

(ajout de ":" pour marquer une pause)

Reply:
and one standard. The only one...

(ajout de".")